### PR TITLE
[ft-164833457] Approve user membership

### PR DESCRIPTION
**Pull request by fridolinho**
## What does this PR do?
Approve user membership
## Description of Task to be completed?
Update user membershipDate to the current date
## How should this be manually tested?

    Postman by sending a patch request 
    route (api/v1/user/:id/approve)

## What are the relevant pivotal tracker stories?
[approve user membership](https://www.pivotaltracker.com/story/show/164833457)
## Screenshots (if appropriate)
image

### CODE REVIEW

File: server/models/users.js [hunk 1]
```diff
@@ -20,6 +20,15 @@ class User {
       return error;
     }
   }
+
+  async approveUser(data) {
```
Comment (niyobobo) [inline server/models/users.js:34]: Solve indentation issue here. (use single tab or double space)

File: server/controllers/users.js [hunk 2]
```diff
@@ -44,6 +44,23 @@ const user = {
       message: 'User not found',
     });
   },
+
+  async approveUser(req, res) {    const {
```
Comment (niyobobo) [inline server/controllers/users.js]: Start object destructuring on a new line.

**Review by niyobobo — COMMENTED** [review]
Work on those feedback

File: server/config/queries.js [hunk 3]
```diff
@@ -1,150 +1,157 @@
...
+	`;
 
 const deleteUser = `
-        DELETE FROM users
-        WHERE userId = $1;
-    `;
+		DELETE FROM users
+		WHERE userId = $1;
+	`;
 const createLoan = `
```
Comment (aaronsekisambu) [inline server/config/queries.js:48]: Please check the ESlint setting and use double spacing.

File: server/controllers/users.js [hunk 4]
```diff
@@ -3,47 +3,65 @@ import userModel from '../models/users';
...
+				message: 'User deleted successfully from members',
+			});
+		}
+		return res.status(404).send({
+			status: res.statusCode,
+			message: 'User not found',
+		});
+	},
+
+	async approveUser(req, res) {
```
Comment (aaronsekisambu) [inline server/controllers/users.js:87]: implement try and catch or then and catch in this controller

**Pull request merged**